ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -201,10 +203,18 @@ def add_remote(self, options):
             self.catalog.add_remote(
                 options['id'], options['name'], options['url'])
 
-        except ValueError as e:
-            raise CommandError(e)
-
-        self.catalog.update_cache()
+            self.catalog.update_cache()
```

Comment:
Just a nit: I'd put this line in an `else` block following the `except one`.

The reason is that only the `add_remote` line may raise an `ExistingRemoteError`, which is made clearer by the `else` block I suggest.

I'd consider it more important if you weren't specifically catching an `ExistingRemoteError` which makes no sense in the context of the `update_cache` method, though, so I wouldn't block the review on this.